examples/python/README.md

```diff
@@ -1,7 +1,18 @@
 ### Pre-requisite
-- have python installed , preferably `>=3.0`
-- install the requirements mentioned in the `requirement.txt`
-- make required changes in the `settings.json` to run the app on custom url & port
+- Have python installed, preferably `>=3.0`
+- Install the requirements mentioned in the `requirement.txt` with `pip3 install -r requirements.txt`
+- Configure the following environment variables to run the app on custom port/url
+  ```
+  export APP_PORT=8080
+  export ROOT_URL=http://localhost:8080
+  export APP_HOST=localhost
```

Comment:
Some PaaS systems require you to use `PORT` as the env variable for the port to listen on. Let's stick with `PORT`

Reply:
done